mithril-stm/src/lib.rs

```diff
@@ -16,7 +16,7 @@ pub use crate::error::{
 };
 
 #[cfg(feature = "benchmark-internals")]
-pub mod multi_sig;
+pub mod bls_multi_signature;
 
 #[cfg(not(feature = "benchmark-internals"))]
-mod multi_sig;
+pub mod bls_multi_signature;
```

Comment:
I guess that we don't want to make public the underlying BLS multi-signature except in the case of the benchmarks:
```suggestion
mod bls_multi_signature;
```